### ISSUE TITLE
Staging Review finding: Link to pay.gov should open in a new tab

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#121234]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Financial Management
**Product:** Debt Portal
**Feature:** Payment History/One thing per page (VHA) MVP

## Findings details
**VA.gov Experience Standard - issue:** [User is taken out of the primary flow without notification.](https://design.va.gov/about/experience-standards/#user-is-notified-if-an-action-will-take-them-out-of-the-primary-flow)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility)

### Description

On the Resolve debt page, we tell Veterans "You’ll need to provide the following details to pay this debt online" once they reach pay.gov. The link to go to pay.gov, however, opens in the same window, taking Veterans away from the page with the information they need to provide for payment.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="726" height="492" alt="Image" src="https://github.com/user-attachments/assets/c8f34303-b343-4622-8000-f7d695e7b3bb" />

</details>

## Recommended action

Use the `external` link component variant which automatically opens the link in a new tab and appends the text `(opens in a new tab)`.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Link component](https://design.va.gov/components/link)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by erinrwhite** [comment]
hi team - following up since the `external` variant is not available for primary action links. I recommend the [Link - Active](https://design.va.gov/storybook/?path=/story/components-va-link--active) component for this change.

**Comment by HeatherRienks** [comment]
This is being addressed on the Financial Management workboard in ticket #121463.

**Comment by Tom-Davis** [comment]
This was addressed in ticket #121463.
Closing this ticket.

@HeatherRienks @denisecoveyduc
